browser/src/dom/NotebookbarAccessibility.js

```diff
@@ -238,6 +239,10 @@ var NotebookbarAccessibility = function() {
 		this.combination = null;
 		this.mayShowAcceleratorInfoBoxes = false;
 		this.filteredItem = null;
+		var accessKeyElements = document.querySelectorAll('.ui-content.unobutton');
+		for (var i = 0; i < accessKeyElements.length; i++) {
+			accessKeyElements[i].removeAttribute('accesskey');
+		}
 	};
```

Comment:
I would use a loop like:

loop (activeTabPointers.contentList.length)
    document.getElementById(activeTabPointers.contentList[i].id).accessKey = null;

And we can check the behaviour from scratch, please correct me if i am wrong. We need something like:
* At start, only tabs' access keys are assigned. Indeed they are never removed.
* When user presses Alt or Alt+Shift (needed on Firefox for example), info boxes are shown (state = 0).
* When user picks a tab's letter, tab's content keys are assigned (state = 1).
* When user picks a command or cancelles the process (possible on 'onblur'), content access keys are removed.

I guess we can check above behaviour and have a clean API hopefully. If you can check and improve my suggestions whereever you see possibilities, that'd be great.

Thanks :)